tests/arch/arm/arm_runtime_nmi/testcase.yaml

```diff
@@ -1,4 +1,5 @@
 tests:
   arch.interrupt.arm.nmi:
     arch_whitelist: arm
+    platform_exclude: qemu_cortex_a53
```

Comment:
This should be handled with `filter: CONFIG_ARMV6_M_ARMV8_M_BASELINE or CONFIG_ARMV7_M_ARMV8_M_MAINLINE` like the rest of (Cortex-M only) `test/arch/arm` tests.

There is already a commit that addresses this in #23636.
https://github.com/zephyrproject-rtos/zephyr/pull/23636/commits/669069118538a091c60267000bcdbafd5fbf13e6